src/main/java/Duke.java

```diff
@@ -58,9 +70,7 @@ public static void request() {
                 String by = line.substring(byStart);
                 tasks[index] = new Deadline(description, by);
                 index++;
-                System.out.println("Got it. I've added this task:");
-                System.out.println(tasks[index-1].toString());
-                System.out.println("Now you have " + index + " tasks in the list");
+                printAddTaskMessage(tasks[index - 1]);
             } else if (line.startsWith("event")) {
                 int descriptionStart = 6;
```

Comment:
i dont rly understand what is happening but its best not to hardcode these values :-)